api/mailer/Util.js

```diff
@@ -0,0 +1,34 @@
+const axios = require('axios');
+
+
+async function validateVerificationEmail(){
+  let status = '';
+  await axios
+    .post('api/Auth/sendVerificationEmail')
+    .then(res =>{
+      status = res.data;
+    })
+    .catch(err => {
+      status = err.data;
+    });
+  return status;
+}
+
+async function generateHashedId(email){
+  let hashedId = '';
+  const toHash = { email : email };
+  await axios
+    .post('http://localhost:8080/api/Auth/generateHashedId', toHash)
```

Comment:
import this at the top
```suggestion
    .post(`${GENERAL_API_URL}`/api/Auth/generateHashedId', toHash)
```